lib/debezium/decimal_test.go

```diff
@@ -86,7 +86,7 @@ func TestEncodeDecimal(t *testing.T) {
 	for _, testCase := range testCases {
 		encodedValue, err := EncodeDecimal(testCase.value, testCase.scale)
 		if testCase.expectedErr == "" {
-			decodedValue := DecodeDecimal(encodedValue, nil, testCase.scale)
+			decodedValue := DecodeDecimal(encodedValue, nil, int(testCase.scale))
```

Comment:
It'd be nice to make make `DecodeDecimal` and `Decimal` use `int16` too.

Reply:
Yeah I tried, it's pretty hard

Reply:
What about int64 then?